api/src/test/java/org/openmrs/module/ugandaemr/dataintegrity/ARTRegimenLinesValidationRulesTest.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.module.ugandaemr.dataintegrity;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openmrs.Patient;
+import org.openmrs.module.dataintegrity.rule.RuleResult;
+import org.openmrs.test.BaseModuleContextSensitiveTest;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class ARTRegimenLinesValidationRulesTest extends BaseModuleContextSensitiveTest {
+
+	protected static final String UGANDAEMR_STANDARD_DATASET_XML = "org/openmrs/module/ugandaemr/include/standardTestDataset.xml";
+
+	ARTRegimenLinesValidationRules artRegimenLinesValidationRules;
```

Comment:
@ssmusoke this throws an exception. Cant tell where it has to be imported from.

Reply:
@slubwama I have deleted the test since the actual validation rule was removed from the implementation